internal/manifest/testdata/oci1.index.zstd-selection.json

```diff
@@ -0,0 +1,66 @@
+{
+    "schemaVersion": 2,
+    "mediaType": "application/vnd.oci.image.index.v1+json",
+    "manifests": [
+        {
+            "mediaType": "application/vnd.oci.image.manifest.v1+json",
+            "digest": "sha256:88cb27f194cce3b06994b4af2dd8a5642f4f294b71bda5b74a3fbb74a9522510",
+            "size": 758,
+            "platform": {
+                "architecture": "amd64",
+                "os": "linux"
+            }
+        },
+        {
+            "mediaType": "application/vnd.oci.image.manifest.v1+json",
+            "digest": "sha256:ac9bbf71dacfad2bfbe48f7d46ca279b1b3b351a6c814db760030015167405b8",
+            "size": 772,
+            "annotations": {
+                "io.github.containers.compression.zstd": "true"
+            },
+            "platform": {
+                "architecture": "amd64",
+                "os": "linux"
+            }
+        },
+        {
+            "mediaType": "application/vnd.oci.image.manifest.v1+json",
+            "digest": "sha256:88cb27f194cce3b06994b4af2dd8a5642f4f294b71bda5b74a3fbb74a9522511",
+            "size": 758,
+            "platform": {
+                "architecture": "arm64",
+                "os": "linux"
+            }
+        },
+        {
+            "mediaType": "application/vnd.oci.image.manifest.v1+json",
+            "digest": "sha256:88cb27f194cce3b06994b4af2dd8a5642f4f294b71bda5b74a3fbb74z9522511",
+            "size": 758,
+            "platform": {
+                "architecture": "arm64",
+                "os": "linux"
+            }
+        },
+        {
+            "mediaType": "application/vnd.oci.image.manifest.v1+json",
+            "digest": "sha256:88cb27f194cce3b06994b4af2dd8a5642f4f294b71bda5b74a3fbb74z9522511",
+            "size": 758,
+            "platform": {
+                "architecture": "s390x",
+                "os": "linux"
+            }
+        },
+        {
+            "mediaType": "application/vnd.oci.image.manifest.v1+json",
+            "digest": "sha256:ac9bbf71dacfad2bfbe48m7d46ca279b1b3b351a6c814db760030015167405b8",
```

Comment:
`m`, and `z` elsewhere in this file, is not a valid hexadecimal digit; that makes it long-term uncertain whether the unit test is testing what we want it to.

---

Having digests that only differ by one latter somewhere in the middle is _seriously_ unhelpful for casual readers of the test. “Oh, the first 8 or so characters match, it looks random, so this looks like the one I’m looking for”.

Please either use completely random values that visually differ, or something _explicitly_ structured like `sha256:aaaa…a` and `sha256:bbb…b`